spec/components/candidate_interface/course_choices_review_component_spec.rb

```diff
@@ -375,6 +375,19 @@
     end
   end
 
+  describe '#application_choices' do
```

Comment:
Keen to know how people feel about this spec. To me it felt easier to test the logic in the `#application_choices` method rather than assert against what is and what is not rendered. Is this a code smell though?

Reply:
Personally, this makes sense to em.